.env

```diff
@@ -39,8 +39,27 @@ SECRET_KEY_BASE=56NtB48ear7+wMSf0IQuWDAAazhpb31qyc7GiyspBP2vh7t5zlCsF5QDv76chXeN
 INDEXER_DISABLE_PENDING_TRANSACTIONS_FETCHER=true
 ETHEREUM_JSONRPC_VARIANT=geth
 ETHEREUM_JSONRPC_HTTP_URL=http://kava:8545
+# uncomment below to have blockscout index public testnet
+# ETHEREUM_JSONRPC_HTTP_URL=https://evm.data-testnet.kava.io
+# ETHEREUM_JSONRPC_WS_URL=wss://wevm.data-testnet.kava.io
+# uncomment below to have blockscout index mainnet
+# ETHEREUM_JSONRPC_HTTP_URL=https://evm.data.kava.io
+# ETHEREUM_JSONRPC_WS_URL=wss://wevm.data.kava.io
+
 NETWORK= "Kava Ethereum Co-Chain"
 LOGO="/images/kava-logo.png"
 LOGO_TEXT="Ethereum Co-Chain"
 COIN=KAVA
 BLOCKSCOUT_VERSION=v5.1.5-kava
+
+# settings used to control blockscout indexer behavior and request rates
+# https://docs.blockscout.com/for-developers/information-and-settings/env-variables#indexer-management
+INDEXER_CATCHUP_BLOCKS_BATCH_SIZE=1
+INDEXER_CATCHUP_BLOCKS_CONCURRENCY=1
+INDEXER_INTERNAL_TRANSACTIONS_CONCURRENCY=1
+INDEXER_BLOCK_REWARD_CONCURRENCY=1
+INDEXER_RECEIPTS_CONCURRENCY=1
+INDEXER_COIN_BALANCES_CONCURRENCY=1
+INDEXER_CATCHUP_BLOCK_INTERVAL=5s
```

Comment:
obvs will set higher / faster values for prod where blockscout can run against more / beefier nodes than a single kava node locally